zigpy/ota/providers.py

```diff
@@ -386,38 +353,18 @@ async def _load_index(
             )
 
 
+# stub provider to keep existing configurations working
 @register_provider
 class Salus(BaseOtaProvider):
     NAME = "salus"
-    MANUFACTURER_IDS = (4216, 43981)
```

Comment:
Due to removing the manufacturer ids, `compatible_with_device` will always return `True` now.

We could either keep the Salus IDs (no images will be loaded), or just use `-1` as an ID, or override `compatible_with_device` with `return False`.

Reply:
I think you can leave them in, since no images will ever be loaded.